docs/getting-started/installation.md

```diff
@@ -10,19 +10,16 @@ In this section you will find an aggregation of the different ways to install Tr
     Add repository setting to `/etc/yum.repos.d`.
 
     ``` bash
-    RELEASE_VERSION=$(grep -Po '(?<=VERSION_ID=")[0-9]' /etc/os-release)
```

Comment:
We replaced `releasever` with the current approach. Please reference the following PR.
https://github.com/aquasecurity/trivy/pull/2143